### PR TITLE
Sound warning

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#49 

#### What does this implement/fix? Explain your changes.
After this PR we will raise a warning after each failing trial for using `playsound`.

### CODE REVIEW

**Comment by sadrasabouri** [comment]
@sepandhaghighi Is there any way we can add tests for this new feature?

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 💯 

1. I suggest calling the `sound_check` function in the `run` function, instead of `CLI`.

2. You can remove `Silence.wav` manually in the test, to check the exception section of `sound_check` function

3. Please update `CHANGELOG.md`

**Comment by sadrasabouri** [comment]
@sepandhaghighi Can you take a look at it again?

File: test/test.py [hunk 1]
```diff
@@ -110,5 +113,10 @@
 >>> play_sound(1, debug=False).join()
 >>> play_sound(1, debug=True).join()
 ERROR : Unable to play sound.
-
+>>> temp = shutil.copyfile("nafas/sounds/silence.wav", "nafas/sounds/temp.wav")
+>>> os.remove("nafas/sounds/silence.wav")
+>>> with warns(RuntimeWarning, match="Your device is not compatible with our underlaying sound playing library. You can refer to https://github.com/sepandhaghighi/nafas/issues/49 if you wanted to solve it."):
+...     sound_check()
+>>> _ = shutil.copyfile(temp, "nafas/sounds/silence.wav")
```
Comment (sepandhaghighi) [inline test/test.py:120]: Why?

Comment (sadrasabouri) [reply test/test.py:120]: For the local run, I think it's better to retrieve the `silence.wav` file.

File: nafas/params.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
 NAFAS_VERSION = "0.6"
 
 SOUND_ERROR_MESSAGE = "ERROR : Unable to play sound."
+SOUND_WARNING_MESSAGE = "Your device is not compatible with our underlaying sound playing library. You can refer to https://github.com/sepandhaghighi/nafas/issues/49 if you wanted to solve it."
```
Comment (sepandhaghighi) [inline nafas/params.py:22]: 1. Did you mean `underlying`?
2. sound-playing
3. Remove this section: `if you wanted to solve it`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: nafas/params.py (hunk 2 above)
Comment (sadrasabouri) [reply nafas/params.py]: Solved in [f0ad4bd2d587](https://github.com/sepandhaghighi/nafas/pull/71/commits/f0ad4bd2d587c4ba75a38c988c9e9b3815754033).

**Review by sepandhaghighi — APPROVED** [review]
LGTM 💯

**Pull request merged**